server/routes/viewingLicences/index.ts

```diff
@@ -22,26 +27,16 @@ export default function Index({ licenceService, prisonerService, qrCodeService }
       asyncMiddleware(handler)
     )
 
-  const post = (path: string, handler: RequestHandler, type?: new () => object) =>
-    router.post(
-      routePrefix(path),
-      roleCheckMiddleware(['ROLE_LICENCE_CA', 'ROLE_LICENCE_RO']),
-      fetchLicence(licenceService),
-      validationMiddleware(type),
-      asyncMiddleware(handler)
-    )
-
-  const viewCasesHandler = new ViewAndPrintCaseRoutes(licenceService)
+  const viewCasesHandler = new ViewAndPrintCaseRoutes(caseloadService)
   const viewLicenceHandler = new ViewAndPrintLicenceRoutes()
   const printHandler = new PrintLicenceRoutes(prisonerService, qrCodeService, licenceService)
-  const comDetailsHandler = new ComDetailsRoutes()
+  const comDetailsHandler = new ComDetailsRoutes(communityService)
 
   get('/cases', viewCasesHandler.GET)
-  post('/cases', viewCasesHandler.POST)
   get('/id/:licenceId/show', viewLicenceHandler.GET)
   get('/id/:licenceId/html-print', printHandler.preview)
   get('/id/:licenceId/pdf-print', printHandler.renderPdf)
-  get('/id/:licenceId/com-details', comDetailsHandler.GET)
+  get('/id/:licenceId/probation-practitioner', comDetailsHandler.GET)
```

Comment:
COMs will only have access to here if they are allowed to view the licence, access control is handled by `fetchLicenceMiddleware`